src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/AnnotatedVariantProducer.java

```diff
@@ -90,7 +90,7 @@ static VariantContext produceAnnotatedVcFromInferredTypeAndRefLocations(final Si
                 .id(inferredType.getInternalVariantId())
                 .attribute(GATKSVVCFConstants.SVTYPE, inferredType.toString());
 
-        if (inferredType instanceof SimpleSVType)
```

Comment:
I don't think there's anything wrong with using `instanceof`.

Reply:
Strangely it failed... Plus it might be better if an alt route exists that doesn't use reflection and has equal line of code works